Lib/multiprocessing/pool.py

```diff
@@ -254,8 +256,8 @@ def _setup_queues(self):
     def apply(self, func, args=(), kwds={}):
         '''
         Equivalent of `func(*args, **kwds)`.
+        self._state should be RUN.
```

Comment:
Hmm. Comments about internal state do not have a place in the docstring of a public method.

Reply:
"Pool must be running"?

Reply:
That sounds ok, though I'm not sure it bears mentioning.